template-compose/app/src/main/java/co/nimblehq/template/compose/ui/theme/AppColors.kt

```diff
@@ -0,0 +1,27 @@
+package co.nimblehq.template.compose.ui.theme
+
+import androidx.compose.material.*
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+// Base colors here
+internal val GreenCitrus = Color(0xFF99CC00)
```

Comment:
Is this redundant? 🙇

Reply:
It's redundant, but I would like to keep a sample color for consistency. Maybe we can comment it as a hint.

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/45b8205f525139e5abc7d3f57859b4212daca3d7